tests/v3_api/test_nfs.py

```diff
@@ -0,0 +1,215 @@
+import pytest
+from .common import *  # NOQA
+
+namespace = {"p_client": None,
+             "ns": None,
+             "cluster": None,
+             "project": None,
+             "pv": None,
+             "pvc": None}
+
+CATTLE_TEST_URL = os.environ.get('CATTLE_TEST_URL', "http://localhost:80")
+ADMIN_TOKEN = os.environ.get('ADMIN_TOKEN', "None")
```

Comment:
@jiaqiluo CATTLE_TEST_URL and ADMIN_TOKEN are already available in common.py . It can be removed from here.